frontend/src/components/table/BookingsTable.tsx

```diff
@@ -46,10 +46,10 @@ export default function BookingsTable({
         columns={[
           {
             render: (row: any) => (
-              <DateTimeCell
+              row.dates?.length > 0 ? <DateTimeCell
```

Comment:
is dates nullable?

Reply:
I found it happened very few time, but maybe the timing is when backend code was not addressed correctly. 

Normally dates nullable are not happening. 

Thinking to add this, just in case.